webcompat/form.py

```diff
@@ -97,6 +99,23 @@ def get_form(ua_header):
     return bug_form
 
 
+def get_details(details_string):
+    """Return details content as a formatted string, if it was JSON. Otherwise,
+    just return the string as-is."""
+    # TODO: file a bug and change DecoderDoctor to encode as JSON so we only
```

Comment:
Actually, this can be simplified now, it already sends JSON:

https://searchfox.org/mozilla-central/source/browser/base/content/browser-media.js#352

Reply:
I guess I forgot I wrote that patch? 😆 

https://reviewboard.mozilla.org/r/216014/diff/2#index_header